jupyter_server/services/contents/handlers.py

```diff
@@ -106,9 +106,13 @@ async def get(self, path=''):
         if content not in {'0', '1'}:
             raise web.HTTPError(400, u'Content %r is invalid' % content)
         content = int(content)
-
+        page = self.get_query_argument('page', default='1')
```

Comment:
It would probably be more backwards compatible if it defaulted to not being paginated if the page argument is not given.